easybuild/easyconfigs/o/OpenMPI/OpenMPI-4.1.1rc2-GCC-10.3.0.eb

```diff
@@ -0,0 +1,35 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+name = 'OpenMPI'
+version = '4.1.1rc2'
+
+homepage = 'https://www.open-mpi.org/'
+description = """The Open MPI Project is an open source MPI-3 implementation."""
+
+toolchain = {'name': 'GCC', 'version': '10.3.0'}
+
+source_urls = ['https://www.open-mpi.org/software/ompi/v%(version_major_minor)s/downloads']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['574ffdccc0cee90c24ae2144d51d2540f3684eac1cd26017081d35b408a3caf4']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('hwloc', '2.4.1'),
+    ('libevent', '2.1.12'),
+    ('UCX', '1.10.0'),
+    ('libfabric', '1.12.1'),
+    ('PMIx', '3.1.6'),
```

Comment:
@robert-mijakovic Any reason for not using PMIx 3.2.3?

Reply:
I have a version with PMIx 3.2.3 and we are actually using it and LuxProvide.
At some point in time, I asked during the meeting about using newer releases and we were supposed to check with OpenMPI guys about their recommendation but I don't think this actually happened. I think sticking to 3.2.3 makes sense as this is a stable branch not an early release like 4.0.0. I'll do it in the morning.